crates/rover-client/src/query/subgraph/list.rs

```diff
@@ -0,0 +1,193 @@
+use crate::blocking::StudioClient;
+use crate::RoverClientError;
+use graphql_client::*;
+
+type Timestamp = String;
+
+#[derive(GraphQLQuery)]
+// The paths are relative to the directory where your `Cargo.toml` is located.
+// Both json and the GraphQL schema language are supported as sources for the schema
+#[graphql(
+    query_path = "src/query/subgraph/list.graphql",
+    schema_path = ".schema/schema.graphql",
+    response_derives = "PartialEq, Debug, Serialize, Deserialize",
+    deprecated = "warn"
+)]
+/// This struct is used to generate the module containing `Variables` and
+/// `ResponseData` structs.
+/// Snake case of this name is the mod name. i.e. list_subgraphs_query
+pub struct ListSubgraphsQuery;
+
+#[derive(Clone, PartialEq, Debug)]
+pub struct SubgraphInfo {
+    pub name: String,
+    pub url: Option<String>, // optional, and may not be a real url
+    pub updated_at: String,
```

Comment:
Is it possible to represent this as a DateTime with types provided by `chrono` rather than a String? This would help us with displaying timezone aware date strings in the output table. This type also implements [Ord](https://docs.rs/chrono/0.4.19/chrono/struct.DateTime.html#impl-Ord) which would allow us to replace `partial_cmp` with `cmp` in our sorting algorithm, which prevents the need for the `unwrap` [here](https://github.com/apollographql/rover/pull/185/files#diff-b2c7ed540735b67729da55b4b400fcbc4b42d4976853262f6ea73bfdf877ae9bR99)